src/components/ItemEditorPage/LeftPanel/LeftPanel.tsx

```diff
@@ -90,6 +94,17 @@ export default class LeftPanel extends React.PureComponent<Props> {
                       onSetCollection={onSetCollection}
                     />
                   )}
+                  <Items
+                    items={items.sort(sortByName)}
```

Comment:
I believe it was already being [sorted by name here](https://github.com/decentraland/builder/pull/1947/files/be06561893372515fe5dd2d4e58791229084442b#diff-d89936af62c59b768318811adc61d46f963afb576d96c3f02ac7990954535049L35), I didn't want to change that. I think you're right, it might be better to remove the sort, right? To avoid confusions between the TP detail page and this list.